ee/api/test/test_authentication.py

```diff
@@ -698,3 +698,10 @@ def test_cannot_use_saml_without_enterprise_license(self):
             str(e.exception),
             "Authentication failed: Your organization does not have the required license to use SAML.",
         )
+
+    def test_xmlsec_and_lxml(self):
```

Comment:
I added a test to avoid accidentally pushing the changes like I did earlier this week

Reply:
Nice call, thank you!